tests/ZichtTest/Bundle/UrlBundle/Controller/SuggestUrlControllerTest.php

```diff
@@ -3,21 +3,34 @@
  * @author Gerard van Helden <gerard@zicht.nl>
  * @copyright Zicht Online <http://zicht.nl>
  */
+
 namespace ZichtTest\Bundle\UrlBundle\Controller;
 
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\HttpFoundation\Request;
+use Zicht\Bundle\UrlBundle\Controller\StaticReferenceController;
+
+/**
+ * Class SuggestUrlControllerTest
+ *
+ * @package ZichtTest\Bundle\UrlBundle\Controller
+ */
 class SuggestUrlControllerTest extends \PHPUnit_Framework_TestCase
 {
+    /**
+     * Test
+     */
     public function testSuggestUrlAction()
     {
-        $controller = new \Zicht\Bundle\UrlBundle\Controller\StaticReferenceController();
-        $container = new \Symfony\Component\DependencyInjection\Container();
+        $controller = new StaticReferenceController();
+        $container = new Container();
```

Comment:
Feel free to do that.  But the changes made here fix a test that is unrelated to this PR, so improvements to this test are out of the scope